Task/Taskfile.yml

```diff
@@ -37,6 +37,55 @@ tasks:
     cmds:
       - pipenv lock
 
+  runner-apt-install:
+    desc: Install something on a GitHub Actions runner via apt
+    internal: true
+    status:
+      - '{{if eq .GITHUB_ACTIONS "true"}}exit 1{{end}}'
+    requires:
+      vars: [TOOLS]
+    platforms: [linux]
+    cmds:
+      - for:
+          var: TOOLS
+          split: ','
+          as: tool
+        cmd: apt-get install -y --no-install-recommends {{.tool}}
+
+  runner-curl-install:
+    desc: Install something on a GitHub Actions runner via curl
+    internal: true
+    status:
+      - '{{if eq .GITHUB_ACTIONS "true"}}exit 1{{end}}'
+    preconditions:
+      - test -d "${RUNNER_TEMP}/bin" || mkdir "${RUNNER_TEMP}/bin"
+      - |
+        case ${GITHUB_PATH} in
+          *${RUNNER_TEMP}/bin*) exit 0 ;;
+          *) echo "${RUNNER_TEMP}/bin" >> "${GITHUB_PATH}" ;;
+        esac
+    requires:
+      vars:
+        - INSTALL_URL
+        - TOOL
+    platforms: [linux]
+    cmds:
+      - curl -sSfL {{.INSTALL_URL}} | sh -s -- -b "${RUNNER_TEMP}/bin"
+      - chmod +x "${RUNNER_TEMP}/bin/{{.TOOL}}"
+
+  mac-brew-install:
+    desc: Install something on a mac via brew
+    internal: true
+    platforms: [darwin]
+    requires:
+      vars: [TOOLS]
+    cmds:
+      - for:
+          var: TOOLS
+          split: ','
+          as: tool
+        cmd: brew install {{.tool}}
```

Comment:
I prefer installing one-at-a-time for troubleshooting and such, but could be convinced to do a `brew install x y z` and cut out the for.  `TOOLS` would then become space-separated instead of , separated.